### PR TITLE
OCTOPUS-696

### PR DESCRIPTION
**Pull request by prb112**

### CODE REVIEW

File: variables.tf [hunk 1]
```diff
@@ -181,16 +181,18 @@ variable "rhel_smt" {
...
   }
+  type        = object({ count = number, memory = string, processors = string })
+  description = "The worker configuration details. You may have 0 or more workers"
   validation {
-    condition     = lookup(var.worker, "count", 1) >= 1
-    error_message = "The worker.count value must be greater than 1."
+    condition     = lookup(var.worker, "count", 1) >= 0
+    error_message = "The worker.count value must be greater than 0."
   }
+  nullable = false
```
Comment (prb112) [inline variables.tf:195]: Revised the order to match the hashicorp doc ordering https://developer.hashicorp.com/terraform/language/values/variables#custom-validation-rules 

Relaxed to 0 and marked nullable = false

File: modules/4_pvs_support/pvs_support.tf [hunk 2]
```diff
@@ -379,3 +379,22 @@ EOF
...
+    timeout     = "${var.connection_timeout}m"
+  }
+
+  provisioner "remote-exec" {
+    inline = [<<EOF
+echo "INFO: number of workers is '${var.worker["count"]}''"
+EOF
+    ]
+  }
+}
```
[the remark this answers is not in the document]
Comment (Chandan-Abhyankar) [reply modules/4_pvs_support/pvs_support.tf:400]: Paul, one thought, Should we add this info/warning code to our first module i.e. checks ?
This way user will get warning early. Thanks

[the remark this answers is not in the document]
Comment (prb112) [reply modules/4_pvs_support/pvs_support.tf:400]: Unfortunately, it requires the null_resource provisioner. We don't have a way to ssh in the fist module.

[the remark this answers is not in the document]
Comment (Chandan-Abhyankar) [reply modules/4_pvs_support/pvs_support.tf:400]: ok, make sense

**Review by Chandan-Abhyankar — APPROVED** [review]
Looks good

**Pull request merged**